src/qtwebkit.mk

```diff
@@ -4,10 +4,10 @@ PKG             := qtwebkit
 $(PKG)_WEBSITE  := https://github.com/annulen/webkit
 $(PKG)_DESCR    := QtWebKit
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 5.13.1
-$(PKG)_CHECKSUM := f688e039e2bdc06e2e46680f3ef57715e1b7d6ea69fd76899107605a8f371ea3
-$(PKG)_SUBDIR   := qtwebkit-everywhere-src-$($(PKG)_VERSION)
-$(PKG)_FILE     := qtwebkit-everywhere-src-$($(PKG)_VERSION).tar.xz
+$(PKG)_VERSION  := 5.212
```

Comment:
This looks like going back to the old version in #2414?

Reply:
qtwebkit-opensource-src-5.212.tar.xz != qtwebkit-everywhere-src-5.13.1.tar.xz

Reply:
And yes, I don't understand why they did this...